src/css/base/_typography.scss

```
@@ -49,7 +49,7 @@ blockquote {
     text-align: right;
 
     p::before {
-      content: "\2014";
+      content: '\2014';
```

This should still be a double quote
```suggestion
      content: "\2014";
```